Directory.Build.targets

```diff
@@ -8,4 +8,12 @@
     <!-- Avoid compile error about missing namespace when combining ImplicitUsings with .NET Framework target frameworks. -->
     <Using Remove="System.Net.Http" Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework'" />
   </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="Strings.resx" EmitFormatMethods="true" />
```

Comment:
💭 Interesting approach putting this in the root file, but I suppose it works.

Reply:
I really like those formatting methods. I want them everywhere. :)

Reply:
But I guess it only applies to the one project, and makes an assumption about the resx file name that only applies to one as well. In my test project I have a different Update item. So I guess I'll move this back into the project.